src/lib.rs

```diff
@@ -481,6 +484,37 @@ pub fn main() -> Result<(), Error> {
             Ok(())
         }
 
+        Commands::Delete {
+            db,
+            genome_ids,
+            output_file,
+        } => {
+            let ref_db = utils::strip_sketch_extension(db);
+
+            log::info!("Reading input genomes");
+            let path = Path::new(genome_ids);
+            let file = File::open(path)?;
+            let reader = std::io::BufReader::new(file);
+
+            // Read in genomes to
+            let ids: Vec<String> = reader.lines().map_while(Result::ok).collect();
+
+            log::info!("Reading input metadata");
+            let mut sketches: MultiSketch = MultiSketch::load(ref_db)
+                .unwrap_or_else(|_| panic!("Could not read sketch metadata from {}.skm", ref_db));
+
+            // write new .skm
+            let _ = sketches.remove_metadata(output_file, &ids);
```

Comment:
I think this can just be:
```suggestion
            sketches.remove_metadata(output_file, &ids);
```
to ignore the return value

Reply:
I had it like this because otherwise the error isn't handled.
unused `Result` that must be used
this `Result` may be an `Err` variant, which should be handled

Reply:
In that case I think you either want to use `.expect()` with an appropriate message to handle the error, or use `?;` at the end to propogate the error to the end (if the whole function returns a `Result` type)